images/mta-analyzer-addon.yml

```diff
@@ -0,0 +1,33 @@
+cachito:
+  packages:
+    gomod:
+      - path: .
+content:
+  source:
+    dockerfile: konflux.Dockerfile
+    git:
+      branch:
+        target: mta-8.0
+      url: git@github.com:openshift-priv/migtools-mta-tackle2-addon-analyzer.git
+      web: https://github.com/migtools/mta-tackle2-addon-analyzer
+distgit:
+  component: mta-analyzer-addon-container
+  branch: rhaos-{MAJOR}.{MINOR}-rhel-9
+delivery:
+  delivery_repo_names:
+  - mta/mta-analyzer-addon-rhel9
+for_payload: false
+enabled_repos:
+- rhel-9-appstream-rpms
+- rhel-9-baseos-rpms
+from:
+  builder:
+    - stream: rhel-9-golang
+  stream: mta-analyzer-lsp
```

Comment:
Comment to acknowledge that mta-analyzer-addon uses mta-analyzer-lsp image as its base image.
_No change if correct._

Reply:
Affirmative, mta-analyzer-addon uses mta-analyzer-lsp as its base runner image

ref here : https://github.com/migtools/konflux-build-data/blob/mta-8.0/mta-analyzer-addon/konflux.Dockerfile#L9